tests/python/unittest/test_numpy_op.py

```diff
@@ -4463,6 +4464,52 @@ def hybrid_forward(self, F, a):
             assert_almost_equal(mx_out.asnumpy(), np_out, rtol=1e-3, atol=1e-5, use_broadcast=False)
 
 
+@with_seed()
+@use_np
+def test_np_full_like():
+    class TestFullLike(HybridBlock):
+        def __init__(self, fill_value, dtype):
+            super(TestFullLike, self).__init__()
+            self._fill_value = fill_value
+            self._dtype = dtype
+
+        def hybrid_forward(self, F, x, *args, **kwargs):
+            return F.np.full_like(x, self._fill_value, self._dtype)
+
+    if StrictVersion(platform.python_version()) < StrictVersion('3.0.0'):
```

Comment:
You can decorate the test function with this condition as the following.
```python
@unittest.skipIf(StrictVersion(platform.python_version()) < StrictVersion('3.0.0'), "reason to skip")
def test_np_full_like():
    ...
```

Reply:
thanks!